web/yo/app/scripts/controllers/gene.js

```diff
@@ -48,8 +48,25 @@ angular.module('oncokbApp')
                     }
                 }
             };
+            var colorsByLeve = {
```

Comment:
You can leave this part in app.js into rootscope. Similar to what we did for the publci repo